Athos/ONNXCompiler/.gitignore

```diff
@@ -0,0 +1,7 @@
+models/
+debug/
+*.cpp
+*.h
+*.ezpc
+*.h
```

Comment:
Also, why is are input files named .h? Can we use something like .inp? Or something more intuitive? This seems confusing to me.

Reply:
done